README.md

```diff
@@ -1,2 +1,42 @@
 # Mbed OS Examples for the ATECC608A
 
+The examples in this repository demonstrate how to use the ATECC608A secure element with Mbed OS. The `atecc608a` example demonstrates use of the ATECC608A with Mbed Crypto. Examples of how to generate a certificate using pre-provisioned ATECC608A keys and how to use ATECC608A with Mbed TLS are yet to come.
+
+## Prerequisites
+
+* [Install Mbed CLI](https://os.mbed.com/docs/mbed-os/latest/tools/installation-and-setup.html).
+
+* [Install the arm-none-eabi-ggc toolchain](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads).
+
+## Building the ATECC608A example
+
+```sh
+git clone git@github.com:ARMmbed/mbed-os-example-atecc608a.git
+cd mbed-os-example-atecc608a/
+source ~/venvs/mbed/bin/activate
+mbed new .
+mbed deploy
+mbed compile -t GCC_ARM -m K64F --flash --sterm
+```
+
+## Hardware interface
+
+A couple of evaluation and development kits are available for the ATECC508A secure element.
```

Comment:
ATECC508A or ATECC608A?

Reply:
The chips are compatible; ATECC608A is a superset of ATECC508A. We don't use ATECC608A-only features. We can say ATECC608A here for consistency.